homepairs/homepairsUI/src/Screens/Components/Modals/EditPropertyModal/EditPropertyModalBase.tsx

```diff
@@ -89,153 +95,118 @@ export default class AddNewPropertyModalBase extends React.Component<Props, Crea
-        containerStyle: {
-            flex: 1,
-            alignSelf: 'center',
-            justifyContent: 'center',
-            minHeight: 50,
-        },
-    };
-
     inputFormProps() : {[id: string] : InputFormProps} {
+        const {address, city, state, bedrooms, bathrooms, tenants} = this.state;
         return {
             streetAddress: {
                 name: signUpStrings.inputForms.address,
                 parentCallBack: this.getFormAddress,
                 formTitleStyle: this.inputFormStyle.formTitle,
                 inputStyle: this.inputFormStyle.input,
+                placeholder: address,
             }, 
             city: {
                 name: signUpStrings.inputForms.city,
                 parentCallBack: this.getFormCity,
                 formTitleStyle: this.inputFormStyle.formTitle,
                 inputStyle: this.inputFormStyle.input,
+                placeholder: city,
             }, 
             state: {
                 name: signUpStrings.inputForms.state,
                 parentCallBack: this.getFormState,
                 formTitleStyle: this.inputFormStyle.formTitle,
                 inputStyle: this.inputFormStyle.input,
+                placeholder: state,
             }, 
             numBed: {
                 name: signUpStrings.inputForms.numBed,
                 parentCallBack: this.getFormNumBed,
                 formTitleStyle: this.inputFormStyle.formTitle,
                 inputStyle: this.inputFormStyle.input,
+                placeholder: bedrooms.toString(),
             }, 
             numBath: {
                 name: signUpStrings.inputForms.numBath,
                 parentCallBack: this.getFormNumBath,
                 formTitleStyle: this.inputFormStyle.formTitle,
                 inputStyle: this.inputFormStyle.input,
+                placeholder: bathrooms.toString(),
             }, 
             maxTenants: {
                 name: signUpStrings.inputForms.maxTenants,
                 parentCallBack: this.getFormMaxTenants,
                 formTitleStyle: this.inputFormStyle.formTitle,
                 inputStyle: this.inputFormStyle.input,
-            }
-        }
+                placeholder: tenants.toString(),
+            },
+        };
     }
     
     render() {
+        const {streetAddress, city, state, numBed, numBath, maxTenants} = this.inputFormProps();
+        const {onChangeModalVisibility} = this.props;
         return <SafeAreaView>
             <ScrollView style = {this.inputFormStyle.modalContainer}>
                 <Card
                     showCloseButton = {true}
-                    title= "Create New Property"
-                    closeButtonPressedCallBack={() => this.props.onChangeModalVisibility(false)}
+                    title= "Edit Property"
+                    closeButtonPressedCallBack={() => onChangeModalVisibility(false)}
                     >
-                    <InputForm {...this.inputFormProps().streetAddress}/>
```

Comment:
I'm going to accept this but we don't need to use the renderInputForm helper function. We could simply define a new function that defines an array of our inputForm props then loops through the array using the map helper method. 

More on this when we pull on Wednesday.